test/common/stats/isolated_store_impl_test.cc

```diff
@@ -15,21 +16,25 @@ namespace Stats {
 
 class StatsIsolatedStoreImplTest : public testing::Test {
 protected:
-  StatsIsolatedStoreImplTest() : pool_(store_.symbolTable()) {}
+  StatsIsolatedStoreImplTest()
+      : symbol_table_(SymbolTableCreator::makeSymbolTable()),
+        store_(std::make_unique<IsolatedStoreImpl>(*symbol_table_)), pool_(*symbol_table_) {}
   ~StatsIsolatedStoreImplTest() override {
     pool_.clear();
-    EXPECT_EQ(0, store_.symbolTable().numSymbols());
```

Comment:
The reason that this file had to be changed is that we had to make the store_ be a unique_ptr so it could be reset, to clear all the symbols from the symbol table. This only matters with real symbol tables.